pyproject.toml

```diff
@@ -24,6 +24,8 @@ packages = [
 python = "^3.9"
 torch = "^2.2.2"
 torchvision = "^0"
+transformers = "^4.46.2"
+accelerate = "^1.1.1"
```

Comment:
it doesn't seem to be imported. do we need it?

Reply:
The `accelerate` module is used internally by the HF RT-DETR implementation and we need to install it, otherwise we receive an error message to do it.